src/cgame/cg_main.cpp

```diff
@@ -158,6 +158,9 @@ Cvar::Cvar<float> cg_motionblurMinSpeed("cg_motionblurMinSpeed", "minimum speed
 Cvar::Cvar<bool> cg_spawnEffects("cg_spawnEffects", "desaturate world view when dead or spawning", Cvar::NONE, true);
 
 static Cvar::Cvar<bool> cg_navgenOnLoad("cg_navgenOnLoad", "generate navmeshes when starting a local game", Cvar::NONE, true);
+static Cvar::Cvar<int> cg_navgenMaxThreads(
```

Comment:
consider using the max threads possible by default. no reason not to.

Reply:
https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency

Reply:
Wow it even works in Nacl. I put the default on hardware_concurrency() - 1 since on Windows at least that's the most you can use without making other apps sluggish.

For the server I am leaving it at 1 background thread by default to respect other services which may be running.

Reply:
We should test that hardware_concurrency works on a Linux and a Mac system though

Reply:
Tested that this API works inside and outside Nacl on Linux and Mac.